Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj

```diff
@@ -97,7 +97,7 @@
     <Compile Include="Tiled\TmxTerrain.cs" />
     <Compile Include="Tiled\TmxTile.cs" />
     <Compile Include="Tiled\TmxTileOffset.cs" />
-    <Compile Include="Tiled\TmxTileset.cs" />
+    <Compile Include="Tiled\TmxTileSet.cs" />
```

Comment:
I was pretty sure I checked that they where the same. I think there's something about the way Windows handles it or maybe the windows version of git. Even though I had renamed the file the change didn't show up in the diff. Anyway, you know what to do. I'll leave it in your capable hands.